src/ngraph_sgcompiler_utils.h

```diff
@@ -48,6 +48,28 @@ inline const ngraph::element::Type& getType(int type) {
   return *ngraphType->second;
 }
 
+inline int getType(const ngraph::element::Type& et) {
```

Comment:
We have a lot of competing type systems at play.  Would it be helpful to give this function a more specific name, such as `getMshadowType` or (even better IMHO) `getMshadowScalarType`?

Reply:
Just to clarify, I'm recommending a change to this function's signature to clarify its *return* type, not its input type.

Its return type is simply `int`, which I assume is a requirement. But unless I'm missing something, it's not really clear which library's scalar type system (mshadow's, etc.) pertains to that returned int.